packages/editor-ui/src/components/ParameterInputHint.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+	<div class="hint" ref="hint" v-if="hint" v-html="hint"></div>
+</template>
+
+<script lang="ts">
+import Vue from "vue";
+
+export default Vue.extend({
+	name: 'InputHint',
+	props: ['hint'],
+	mounted(){
+		if(this.$refs.hint){
+			(this.$refs.hint as Element).querySelectorAll('a').forEach(a => a.target = "_blank");
+		}
+	},
+});
+</script>
+
+<style scoped lang="scss">
+.hint {
+	margin-top: var(--spacing-2xs);
+	color: var(--custom-font-very-light);
```

Comment:
does this token actually exist? I can see a similar looking scss variable but I don't think this is applying anything

Reply:
--font-size-3xs I did add it to match Max design, approximately 10px, 0.6rem

Reply:
i am asking about `--custom-font-very-light`

Reply:
line 12 $--custom-font-very-light: #999; in n8n-theme-variables.scss

Reply:
ya that's a scss variable.. that only applies at build time. you are using css variable. this applies at runtime.

Reply:
I see, anyway it's no longer in use since I did replace it with n8n-text component